persistence/sql/persister_consent.go

```diff
@@ -438,3 +438,17 @@ func (p *Persister) VerifyAndInvalidateLogoutRequest(ctx context.Context, verifi
 		return nil
 	})
 }
+
+func (p *Persister) FlushInactiveLoginConsentRequests(ctx context.Context, notAfter time.Time) error {
+	var lr consent.LoginRequest
+	err := p.Connection(ctx).RawQuery(
+		fmt.Sprintf("DELETE FROM %s WHERE requested_at < ? AND requested_at < ?", (&lr).TableName()),
+		time.Now().Add(-p.config.ConsentRequestMaxAge()),
+		notAfter,
+	).Exec()
+
+	if err == sql.ErrNoRows {
+		return errors.Wrap(fosite.ErrNotFound, "")
```

Comment:
```suggestion
		return errors.WithStack(fosite.ErrNotFound)
```

Reply:
Actually, as this is not called by fosite, you don't have to return a fosite error here. Just returning `sqlcon.HandleError(err)` should be fine IMO.